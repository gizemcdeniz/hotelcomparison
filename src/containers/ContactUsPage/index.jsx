import React from "react";
import { Container } from "react-bootstrap";
import { useTranslation } from "react-i18next";
import Contact from "../../components/Contact";
import "./index.scss";

const ContactUsPage = () => {
  const { t } = useTranslation();

  return (
    <Container className="contactUsPage">
      <h2>{t("contactUs.contactUsPage.header")}</h2>
      <h6>{t("contactUs.contactUsPage.subHeader")}</h6>
      <Contact />
    </Container>
  );
};

export default ContactUsPage;
