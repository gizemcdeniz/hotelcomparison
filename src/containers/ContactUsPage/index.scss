@import "../../style/variables";

.contactUsPage {
  .contactContainer {
    background-color: $lightColor;

    .contactForm {
      background-color: $warningColor;
      padding: 3rem;
    }

    .contactInfo {
      h2 {
        text-align: left;
        margin-top: 1rem;
        font-weight: 500;
        font-size: 1.8rem;
      }
    }
  }

  h2 {
    color: $primaryColor;
    font-family: $mainFontFamily;
    margin-top: 4rem;
    text-align: center;
    font-size: 2rem;
    font-weight: bold;
  }

  h6 {
    color: $darkColor;
    text-align: center;
    font-family: $mainFontFamily;
  }
}
