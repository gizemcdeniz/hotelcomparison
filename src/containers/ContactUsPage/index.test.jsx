import React from "react";
import renderer from "react-test-renderer";
import ContactUsPage from "./index";

it("renders correctly and matches the snapshot", () => {
  const tree = renderer.create(<ContactUsPage />).toJSON();
  expect(tree).toMatchSnapshot();
});
