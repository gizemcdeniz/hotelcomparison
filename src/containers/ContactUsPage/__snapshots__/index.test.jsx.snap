// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly and matches the snapshot 1`] = `
<div
  className="contactUsPage container"
>
  <h2>
    contactUs.contactUsPage.header
  </h2>
  <h6>
    contactUs.contactUsPage.subHeader
  </h6>
  <section
    className="contactSection"
  >
    <div
      className="contactContainer container"
    >
      <div
        className="row"
      >
        <div
          className="col-md-6 col-sm-12"
        >
          <div
            className="contactInfo col"
          >
            <h2>
              contactUs.contactUsInfo.header
            </h2>
            <p>
              contactUs.contactUsInfo.description
            </p>
            <div
              className="contactMethods row"
            >
              <div
                className="col"
              >
                <img
                  alt="Location icon"
                  className="contactIcon"
                  src="locationIcon.svg"
                />
                <p>
                  contactUs.contactUsInfo.location
                </p>
              </div>
              <div
                className="col"
              >
                <img
                  alt="Phone icon"
                  className="contactIcon"
                  src="phoneIcon.svg"
                />
                <p>
                  contactUs.contactUsInfo.phone
                </p>
              </div>
              <div
                className="col"
              >
                <img
                  alt="Contact icon"
                  className="contactIcon"
                  src="mailIcon.svg"
                />
                <p>
                  contactUs.contactUsInfo.email
                </p>
              </div>
            </div>
          </div>
        </div>
        <div
          className="col"
        >
          <div
            className="contactForm container-fluid"
          >
            <form
              onSubmit={[Function]}
            >
              <div
                className="formInput"
              >
                <label
                  htmlFor="fullName"
                >
                  contactUs.contactUsForm.name.label
                  <input
                    aria-label="Full Name"
                    id="fullName"
                    name="fullName"
                    placeholder="contactUs.contactUsForm.name.placeholder"
                    type="text"
                  />
                </label>
              </div>
              <div
                className="formInput"
              >
                <label
                  htmlFor="email"
                >
                  contactUs.contactUsForm.email.label
                  <input
                    aria-label="Email address"
                    id="email"
                    name="email"
                    placeholder="contactUs.contactUsForm.email.placeholder"
                    type="email"
                  />
                </label>
              </div>
              <div
                className="formInput"
              >
                <label
                  htmlFor="messageContent"
                >
                  contactUs.contactUsForm.message.label
                  <textarea
                    aria-label="message"
                    id="messageContent"
                    name="messageContent"
                    placeholder="contactUs.contactUsForm.message.placeholder"
                  />
                </label>
              </div>
              <button
                className="submitBtn"
                type="submit"
              >
                contactUs.contactUsForm.submitBtn
              </button>
            </form>
          </div>
        </div>
      </div>
    </div>
  </section>
</div>
`;
